class-wp-bootstrap-navwalker.php

```diff
@@ -211,26 +215,38 @@ public static function fallback( $args ) {
 				$menu_class = $args['menu_class'];
 				$menu_id = $args['menu_id'];
 
+				// initialize var to store fallback html.
+				$fallback_output = '';
+
 				if ( $container ) {
-					echo '<' . esc_attr( $container );
+					$fallback_output = '<' . esc_attr( $container );
 					if ( $container_id ) {
-						echo ' id="' . esc_attr( $container_id ) . '"';
+						$fallback_output = ' id="' . esc_attr( $container_id ) . '"';
 					}
 					if ( $container_class ) {
-						echo ' class="' . sanitize_html_class( $container_class ) . '"'; }
-					echo '>';
+						$fallback_output = ' class="' . sanitize_html_class( $container_class ) . '"';
+					}
+					$fallback_output = '>';
 				}
-				echo '<ul';
+				$fallback_output = '<ul';
 				if ( $menu_id ) {
-					echo ' id="' . esc_attr( $menu_id ) . '"'; }
+					$fallback_output = ' id="' . esc_attr( $menu_id ) . '"'; }
 				if ( $menu_class ) {
-					echo ' class="' . esc_attr( $menu_class ) . '"'; }
-				echo '>';
-				echo '<li><a href="' . esc_url( admin_url( 'nav-menus.php' ) ) . '" title="">' . esc_attr( 'Add a menu', '' ) . '</a></li>';
-				echo '</ul>';
+					$fallback_output = ' class="' . esc_attr( $menu_class ) . '"'; }
+				$fallback_output = '>';
+				$fallback_output = '<li><a href="' . esc_url( admin_url( 'nav-menus.php' ) ) . '" title="">' . esc_attr( 'Add a menu', '' ) . '</a></li>';
+				$fallback_output = '</ul>';
 				if ( $container ) {
-					echo '</' . esc_attr( $container ) . '>'; }
-			}
+					$fb_output = '</' . esc_attr( $container ) . '>';
```

Comment:
This line was missed when the others were changed to the new variable name. 

Needs corrected.